Tests/test_PDB.py

```diff
@@ -1332,6 +1332,25 @@ def test_NACCESS_asa_file(self):
             naccess = process_asa_data(asa)
         self.assertEqual(len(naccess), 524)
 
+class TestSelf(unittest.TestCase):
+    
+    def test_example(self):
+        from Bio.PDB.PDBParser import PDBParser
+        import sys
+        p = PDBParser(PERMISSIVE=True)
+        s = p.get_structure("test", sys.argv[1])
+        io = PDBIO()
+        io.set_structure(s)
+        io.save("out1.pdb")
+        with open("out2.pdb", "w") as fp:
+            s1 = p.get_structure("test1", sys.argv[1])
```

Comment:
There won't be any command line arguments when run via the test framework. You should see if the test make sense with some of the example files we include under ``Tests/PDB/`` here.